vllm/model_executor/model_loader/loader.py

```diff
@@ -386,7 +386,7 @@ def load_model(self, vllm_config: VllmConfig) -> nn.Module:
 
             for _, module in model.named_modules():
                 quant_method = getattr(module, "quant_method", None)
-                if quant_method is not None:
+                if isinstance(quant_method, QuantizeMethodBase):
```

Comment:
Why was this change needed rather than just a None check?

Reply:
When iterating the `named_modules` on intel platforms, a customed layer in intel IPEX actually doesn't have a valid `quant_method` even it's not `None`. So I tighten up this check here to skip processing it which is not necessary.